### PR TITLE
Reducing logging

### PR DESCRIPTION
**Pull request by alfetopito**
As title states

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr222--dexreact.review.gnosisdev.com

File: src/api/wallet/WalletApiImpl.ts [hunk 1]
```diff
@@ -248,7 +248,6 @@ export class WalletApiImpl implements WalletApi {
   private async _notifyListeners(): Promise<void> {
     await Promise.resolve()
     const walletInfo: WalletInfo = this._getWalletInfo()
-    console.table(walletInfo)
```
Comment (Velenir) [inline src/api/wallet/WalletApiImpl.ts]: But, but we never get to use `console.table` :cry:

Comment (W3stside) [reply src/api/wallet/WalletApiImpl.ts]: @Velenir I always know when u find some new thing when it pops up EVERYWHERE lol.

Comment (alfetopito) [reply src/api/wallet/WalletApiImpl.ts]: I'm happy to keep it if we can manage to log it ONLY when it changes

**Pull request merged**